backend/app/views.py

```diff
@@ -16,3 +23,26 @@ class ListUser(generics.ListAPIView):
 class GetUpdateDeleteUser(generics.RetrieveUpdateDestroyAPIView):
   queryset = AppUser.objects.all()
   serializer_class = UserSerializer
+
+
+class LoginUser(APIView):
+  def post(self, request):
+    email = request.data["email"]
+    password = request.data["password"]
+
+    if email and password:
+      user = authenticate(request=request, username=email, password=password)
+
+      if user:
+        login(request, user)
+        return Response(None, status=status.HTTP_200_OK)
+      else:
+        return Response(None, status=status.HTTP_401_UNAUTHORIZED)
```

Comment:
Can this be optimized into 1 return statement? Maybe store the status into a variable then assign them with respect to the condition. Something like:
```
status=null

if user:
  login()
  status = ok
else:
  status = notOk

return Response(None, status=status)